### DISCUSSION TITLE
Default value for attribute is not being set on the appwrite console

### DISCUSSION THREAD

**Discussion by yagyesh-bobde** [appwrite/appwrite#5613] [Q&A] [answered]
I tried giving a default value to an attribute but I wasn't saved in the docuement.

https://github.com/appwrite/appwrite/assets/90238742/06008ac0-b4d5-447a-bd7c-61b7ac5175e8

**Comment by byawitz** [comment, marked as answer]
This was previously noted. Appwrite clouds doesn't use the latest version of Appwrite yet. 
That means till the next update of the cloud it won't work.

**Comment by yagyesh-bobde** [comment]
Alright Thanks!
